src/api/handlers/userprofile.py

```diff
@@ -36,28 +36,31 @@ def registerRoutes(self):
         self.add("/users.households.list", self.list_households)
         self.add("/users.events.list", self.list_events)
         self.add("/users.checkImported", self.check_user_imported)
-        self.add("/users.listForPublicView", self.list_publicview) # NOT USED
+        self.add("/users.listForPublicView", self.list_publicview)  # NOT USED
         self.add("/users.validate.username", self.validate_username)
 
         # admin routes
         self.add("/users.listForCommunityAdmin", self.community_admin_list)
         self.add("/users.listForSuperAdmin", self.super_admin_list)
         self.add("/users.import", self.import_users)
         self.add("/users.mou.accept", self.accept_mou)
- 
+
         self.add("/users.get.visits", self.fetch_user_visits)
-    
-    @admins_only 
-    def fetch_user_visits(self,request): 
-      context: Context = request.context
-      args: dict = context.args
-      args, err = self.validator.expect("id", str, is_required=True).verify(context.args)
-      if err:
-        return err
-      visits, err = self.service.fetch_user_visits(context, args)
-      if err:
-        return err
-      return MassenergizeResponse(data=visits)
+        self.add("/users.listNoPagination", self.users_list_no_pagination)
```

Comment:
same comment as above; I think a URL parameter should accomplish this, not a dedicated route.